### PR TITLE
third commit

### PR DESCRIPTION
**Pull request by AbhiTagline**
Question 3

### CODE REVIEW

File: question3.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sentences = ["My name is Ram", "He is a good person", "You should be careful while coding", "He can do better", "The person is mysterious", "Jay Shree Ram", "It is my pen"]
+
+word_tree = [z for x in [y.split(" ") for y in sentences] for z in x]
+print(word_tree)
+
+dict = {x:word_tree.count(x) for x in word_tree }
```
Comment (himanshu-suthar-tagline) [inline question3.py]: use python `collections.Counter` module.

Comment (AbhiTagline) [reply question3.py]: Why, is there any particular reason?

Comment (himanshu-suthar-tagline) [reply question3.py]: Learn it and answer me the question you just asked me.

Comment (AbhiTagline) [reply question3.py]: Ok, got it.

Comment (himanshu-suthar-tagline) [reply question3.py]: Please update the code to include the collections.Counter.

Comment (AbhiTagline) [reply question3.py]: Done

**Pull request merged**